week06_policy_based/atari_wrappers.py

```diff
@@ -280,11 +280,10 @@ def __init__(self, env, prefix=None, running_mean_size=100, step_var=None):
 
         import tensorflow as tf
         self.step_var = (step_var if step_var is not None
-                         else tf.train.get_global_step())
+                         else tf.summary.experimental.get_step())
 
     def add_summary_scalar(self, name, value):
-        import tensorflow as tf
-        tf.contrib.summary.scalar(name, value, step = self.step_var)
+        tf.summary.scalar(name, value, step=self.step_var)
```

Comment:
I'm pretty sure that `tf` has to be re-imported here. Just tried out something similar on a simpler example:
```{python}
class A:
  def __init__(s): import numpy
  def foo(s): print(numpy)
```
and calling `A().foo()` gives an error.

Reply:
> I'm pretty sure that `tf` has to be re-imported here. Just tried out something similar on a simpler example:
> 
> ```
> class A:
>   def __init__(s): import numpy
>   def foo(s): print(numpy)
> ```
> 
> and calling `A().foo()` gives an error.

Oh, yeah, I was wrong, I'll revert the import.